bevy_matchbox/Cargo.toml

```diff
@@ -25,3 +25,7 @@ ggrs = ["matchbox_socket/ggrs"]
 [dependencies]
 bevy = { version = "0.10", default-features = false }
 matchbox_socket = { path = "../matchbox_socket", version = "0.6" }
+cfg-if = "1.0"
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+matchbox_signaling = { path = "../matchbox_signaling", version = "0.6" }
```

Comment:
This should probably be put behind a non-default feature flag? I imagine most use-cases don't need the peers to act as signaling servers.

Reply:
I feel like there was some reason I didn't do this, but I can't for the life of me remember. Will try sticking it behind a feature and see how it plays out

Reply:
Yeah, no clue why I didn't do that :sweat_smile: